### PR TITLE
Align with changes from vscode-languageclient-7.0.0-next.10

### PR DESCRIPTION
**Pull request by kjeremy**

### CODE REVIEW

**Comment by kjeremy** [comment]
I've tested this against rust-analyzer.

File: src/call_hierarchy.rs [hunk 1]
```diff
@@ -68,6 +69,9 @@ pub struct CallHierarchyItem {
     /// The range that should be selected and revealed when this symbol is being picked, e.g. the name of a function.
     /// Must be contained by the [`range`](#CallHierarchyItem.range).
     pub selection_range: Range,
+
+    /// A data entry field that is preserved between a call hierarchy prepare and incloming calls or outgoing calls requests.
+    pub data: Option<Value>,
```
Comment (Marwes) [inline src/call_hierarchy.rs:74]:
```suggestion
    #[serde(skip_serializing_if = "Option::is_none")]
    pub data: Option<Value>,
```

**Review by Marwes — APPROVED** [review]

**Comment by kjeremy** [comment]
@Marwes I think the build is stuck.

**Comment by kjeremy** [comment]
Should be good to go now.

**Comment by Marwes** [comment]
👍

**Pull request merged**